test/mpi/maint/jenkins/xfail.conf

```diff
@@ -106,6 +106,9 @@
 # freebsd failures
 * * debug ch3:tcp freebsd64     /^comm_create_group_threads/     xfail=issue4372        test/mpi/threads/comm/testlist
 
+# timeout due to lack of passive progress
+* * vci ch4:ofi *       /^rma_contig.*iter=32768/        xfail=ticket0           test/mpi/rma/testlist
```

Comment:
It would still be good to have a ticket number here instead of ticket0.

Reply:
Fixed. https://github.com/pmodels/mpich/issues/5565